conda-env-mod

```diff
@@ -129,8 +128,14 @@ function create_env()
 	#
 	# Now the actual conda create
 	#
-	echo conda create -q --copy "${CONDA_ARGS[@]}" python="${PY}" "${CONDA_XTRA[@]}"
-	conda create -q --copy "${CONDA_ARGS[@]}" python="${PY}" "${CONDA_XTRA[@]}"
+	# Check yaml file
+	if [[ $Opt_YAML -ne 0 ]]; then
+		echo conda env create -q "${CONDA_ARGS[@]}" python="${PY}" "${CONDA_XTRA[@]}"
+		conda env create -q "${CONDA_ARGS[@]}" python="${PY}" "${CONDA_XTRA[@]}"
+	else
+		echo conda create -q --copy "${CONDA_ARGS[@]}" python="${PY}" "${CONDA_XTRA[@]}"
```

Comment:
when you are creating an environment from a specfile, do you still need to specify `python=xxxx`? It may already be in the YAML or spec file.